code/modules/events/visitor.dm

```diff
@@ -0,0 +1,10 @@
+/datum/event/visitor
+    no_fake = 1
+    var/datum/ghostspawner/human/visitor/spawner
+
+/datum/event/rescue_pod/setup()
```

Comment:
Should this be /datum/event/visitor/setup()?

Same for start() below.

Reply:
yes, me tired, good catch